examples/layers/raw/text.dart

```diff
@@ -49,19 +54,36 @@ void beginFrame(Duration timeStamp) {
 }
 
 void main() {
+  // To create a paragraph of text, we use ParagraphBuilder.
   ui.ParagraphBuilder builder = new ui.ParagraphBuilder()
+    // We first push a style that turns the text blue.
     ..pushStyle(new ui.TextStyle(color: const ui.Color(0xFF0000FF)))
     ..addText("Hello, ")
+    // The next run of text will be bold.
     ..pushStyle(new ui.TextStyle(fontWeight: ui.FontWeight.bold))
     ..addText("world. ")
+    // The pop() command signals the end of the bold styling.
     ..pop()
+    // We add text to the paragraph in logical order. The paragraph object
+    // understands bi-directional text and will compute the visual ordering
+    // during layout.
     ..addText("هذا هو قليلا طويلة من النص الذي يجب التفاف .")
+    // The second pop() removes the blue color.
     ..pop()
+    // We can add more text with the default styling.
     ..addText(" و أكثر قليلا لجعله أطول. ");
```

Comment:
Add the following to show that you can go LTR after the RTL:
`..addText('สวัสดี'); // "Hello" in Thai`